.github/workflows/roxy.yml

```diff
@@ -0,0 +1,66 @@
+name: "Deploy MarkLogic config"
+
+on:
+  # Change from push to workflow_dispatch once merged (only available via website once on main branch)
+  # workflow_dispatch:
+  #   inputs:
+  #     git_ref:
+  #       description: 'Commit or branch to deploy'
+  #       required: true
+  #       type: string
+  #     environment:
+  #       description: 'Environment to run against'
+  #       required: true
+  #       type: environment
+
+  push:
+    branches:
+      - "DT-58"
+
+jobs:
+  get_terraform_outputs:
+    name: "Get Terraform outputs"
+    runs-on: ubuntu-latest
+    # unfortunately this means a reviewer needs to approve each job, even when a reviewer triggers the job 
```

Comment:
I don't think I understand this comment?

Reply:
Even when triggering the job manually though GitHub, you still then have to approve each of these steps before they'll run, because the staging environment requires approval.

I think I can make this slightly less annoying by cutting out the first job, but I'll save that for another PR.